src/components/Slider/Slider.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import { Swiper, SwiperSlide } from 'swiper/react';
-import { Autoplay, Navigation } from 'swiper';
+import SwiperCore, { Autoplay, Navigation } from 'swiper';
 import { Container } from '@mui/material';
 import { SliderCard } from './components/SliderCard';
 import { SliderArrow } from './components/SliderArrow';
 import type { Slides } from './types/SliderTypes';
 import { slides } from './mocks/slidesData';
 import { ArrowDirection } from './constants/constants';
-import 'swiper/css';
+import 'swiper/swiper-bundle.css';
```

Comment:
why do you import all bundle? Can we use only CSS for autoplay and navigation

Reply:
I don`t need css for auto-play and navigation modules. But I need styles for slider structure for this works. But in 6.0.2v of swiper we have only one css file to import :(  , so i had used it.